package.json

```diff
@@ -244,6 +244,7 @@
     "@metamask/network-controller@npm:^17.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
```

Comment:
Are these older resolutions still needed?

Reply:
17 and 20 not anymore I can't find them in the lock file. I will remove their resolution